src/overlays/actors/ovl_En_Hit_Tag/z_en_hit_tag.h

```diff
@@ -7,9 +7,11 @@ struct EnHitTag;
 
 typedef void (*EnHitTagActionFunc)(struct EnHitTag*, GlobalContext*);
 
+#define ENHITTAG_GET_SWITCHFLAG(thisx) (s32)(thisx->actor.params & 0xFE00) >> 9)
```

Comment:
```suggestion
#define ENHITTAG_GET_SWITCHFLAG(thisx) (s32)(((thisx)->params & 0xFE00) >> 9)
```
Currently our style, though admittedly inconsistent, is to have these macros take the direct actor pointer. Thus usually called like `ENHITTAG_GET_SWITCHFLAG(&this->actor)`. Its also safer to put the macro arg in parens.